buf.yaml

```diff
@@ -0,0 +1,167 @@
+version: v1beta1
+name: buf.build/grpc-ecosystem/grpc-gateway
+deps:
+  - buf.build/beta/googleapis
+build:
+  roots:
+    - .
+lint:
+  use:
+    - DEFAULT
+  ignore_only:
```

Comment:
These lint ignores are based on existing transgressions. It will fail for new errors.